awx/main/credential_plugins/tss.py

```diff
@@ -49,7 +49,10 @@ def tss_backend(**kwargs):
     secret_dict = secret_server.get_secret(kwargs['secret_id'])
     secret = ServerSecret(**secret_dict)
 
-    return secret.fields[kwargs['secret_field']].value
+    if isinstance(secret.fields[kwargs['secret_field']].value, str) == False:
```

Comment:
Apologies, I wasn't planning on reviewing this myself but in the spirit of helping you get this merged here are my comments. 

There are lots of things that aren't strings. Do we want to do something more like:
```
If <it is a string>:
    return value
elif <its a requests.models.Response object>:
    return value.text
else:
    raise SomeException('Expected string or Response got something else')
    - or -
    return None (or '' or whatever would make sense here)
```

Also, I know we didn't do it before but do we want to be more defensive about ensuring that `kwargs['secret_field']` is a value and it does exist in the `secret.fields`? Since this wasn't done before I wouldn't be offended if we don't want to go down this route.

Reply:
As testing is challenging if change again I need test again which will take time. These are good suggestion definitely. However wanted to keep change as minimal as possible. Can we do future PR to include enhancement and continue with current approach?

Reply:
@john-westcott-iv any updates when this PR reviewed further. As stated earlier I provided this fix as part of a defect found in RedHat product during RedHat customer implementation and got the fix. Its our generosity that I would like to help RH to fix the bug and get permanent fix quicker. Let me know what else can be done.

Reply:
We discussed this yesterday and are going to move forward with this.

Reply:
Thank you 🙏

Reply:
@john-westcott-iv - Thank you for approving the PR. I am not seeing any option to merge it. As its my first time on this branch do I have access for merge or it will done from awx maintainer Team.